draft-ietf-add-ddr.md

```diff
@@ -523,6 +523,66 @@ IANA is requested to add an entry in "Transport-Independent Locally-Served
 DNS Zones" registry for 'resolver.arpa.' with the description "DNS Resolver
 Special-Use Domain", listing this document as the reference.
 
+## Domain Name Reservation Considerations
+
+In accordance with section 8.1 of {{?RFC6761}}, the answers to the following
```

Comment:
```suggestion
In accordance with {{Section 5 of RFC6761}}, the answers to the following
```

Reply:
*sigh* well _that_ would explain the failures. Apparently too many edits in one sprint. Thanks for catching this.